Open-Rival/include/Unit.h

```diff
@@ -130,8 +64,6 @@ namespace Rival {
          */
         int y;
 
-        Facing facing = Facing::South;
-
```

Comment:
My instinct was that `facing` *should* be part of the Unit class. But thinking about it, I don't think it has any significance beyond rendering, so putting it in the animation makes sense, even if it feels a little unnatural to me.